tests/foreman/performance/common/perf_pulp.py

```diff
@@ -0,0 +1,151 @@
+"""Test utilities for writing Pulp tests
+
+Part of functionalities of Pulp are defined in this module
+and have utilities of single repository synchronization, single
+sequential repository sync, sequential repository re-sync.
+
+"""
+
+
+from robottelo.cli.repository import Repository
+
+from perf_standup import StandupPerfTest
+import logging
+
+
+class PulpPerfTest:
+    """Performance Measurement of RH Satellite 6
+
+    Pulp Synchronization functionality
+    """
+
+    # parameters
+    org_id = 1
+
+    logger = logging.getLogger('robottelo')
+    logger.setLevel(logging.DEBUG)
+
+    @classmethod
+    def get_repo_ids_names(cls):
+        """Retrieve list of enabled reposotiry names and ids"""
+        # fetch enabled repository list
+        result = Repository.list(
+            {'organization-id': cls.org_id},
+            per_page=False
+        )
+
+        # repo_ids_list would contain all ids in the hammer repo list
+        repo_ids_list = [repo['id'] for repo in result.stdout]
+        repo_names_list = [repo['name'] for repo in result.stdout]
+        return (repo_ids_list, repo_names_list)
+
+    @classmethod
+    def repository_single_sync(cls, repo_id, repo_name):
+        """Single Synchronization
+
+        @para: repo_id
+        @para: repo_name
+        @return: time measure for a single sync
+
+        note: 7Kickstart repo_id is 5
+        """
+        # perform single sync
+        cls.logger.info("Synchronize {}:".format(repo_name))
+
+        result = Repository.synchronize({
+            'id': repo_id
+        })
+
+        if result.return_code == 0:
+            cls.logger.info("Sync repository {} successful!"
+                            .format(repo_name))
+            return cls.get_elapsed_time(result.stderr)
+        else:
+            cls.logger.error("Sync repository {} failed!".format(repo_name))
+            cls.logger.error(result.stderr)
```

Comment:
Every time the ssh command finds content on stderr it will automatically log that for you. You can remove this and all other stderr logging.